_maps/ministation.json

```diff
@@ -0,0 +1,11 @@
+{
+    "map_name": "MiniStation",
```

Comment:
Try copying and pasting all of this into next_map.json (might be nextmap.json) While loading up let us know if it forces default to box.

Reply:
Turns out map vote was working fine, just can't vote for the current map. Thanks.